lte/gateway/c/oai/tasks/nas/emm/ServiceRequestHdl.c

```diff
@@ -121,11 +121,6 @@ static int emm_service_reject(mme_ue_s1ap_id_t ue_id, uint8_t emm_cause)
   rc = emm_sap_send(&emm_sap);
 
   // Release EMM context
```

Comment:
remove comment? also in this case is not necessary to remove the emm context anymore?

Reply:
Since Service Request comes in S1ap: Initial Ue message, On certain condition when mme sends Service Reject, mme releases the newly established s1-signalling connection, but it should not release the emm context as UE remains in registered state.

Reply:
can you remove the comment then?